src/Hooks/Event.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+namespace Carnival\Hooks;
+
+interface Event { }
```

Comment:
I like `Event` in theory, but I don't like that it would conflate with Laravel's [existing Event concept](https://laravel.com/docs/10.x/events).

We should brainstorm other names for our use case. Do you have any ideas?

Reply:
I agree, but I don't have any ideas on an alternate name.

Reply:
* Topic
* Subscribable
* Subject

Thoughts? Or your own ideas?

Reply:
I like Topic, we could go with that.